kotlin-js-store/yarn.lock

```diff
@@ -0,0 +1,2738 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Must this file checked in ?

Reply:
It will be created again , I'll suggest checking it in

Reply:
https://stackoverflow.com/questions/39990017/should-i-commit-the-yarn-lock-file-and-what-is-it-for